.github/workflows/continuous-integration-optional.yml

```diff
@@ -0,0 +1,134 @@
+name: CI (optional checks)
```

Comment:
I tried to not change the previous behaviour.

Currently, these changes are optional because they don't block merging a PR (repository settings) and don't block a deployment (not part of ci-passed event).

If I would include them in the main CI action, they would not be optional anymore, because the deployments call continuous-integration.yml as a sub-workflow and requires all these tests to pass.

We can also choose to make these tests mandatory (no problem with that). It's just a change from the previous behaviour.

Reply:
Ah, I see, and in the ci-passed event sending code we previously already had a hardcoded [list of all required status checks](https://github.com/ecamp/ecamp3/blob/28b39050ce1d900ff890fdfe2a6a9fa0b23ca4c1/.github/workflows/continuous-integration.yml#L389..L395) in code. So my argument is null and void. Fine for me.